packages/aws-rfdk/lib/lambdas/nodejs/lib/configure-spot-event-plugin/test/spot-event-plugin-client.test.ts

```diff
@@ -238,4 +260,109 @@ describe('SpotEventPluginClient', () => {
     // THEN
     await expect(promise).rejects.toEqual(statusMessage);
   });
+
+  test('successful getCollection for groups', async () => {
+    // WHEN
+    // eslint-disable-next-line dot-notation
+    spotEventPluginClient['deadlineClient'].GetRequest = jest.fn().mockResolvedValue(successfulGroupResponse);
+    const result = await spotEventPluginClient.getCollection(CollectionType.Group);
+
+    // THEN
+    expect(result).toEqual(deadlineGroups);
+    // eslint-disable-next-line dot-notation
+    expect(spotEventPluginClient['deadlineClient'].GetRequest).toBeCalledTimes(1);
+    expect(consoleLogMock.mock.calls.length).toBe(1);
+    expect(consoleLogMock.mock.calls[0][0]).toMatch(/Getting group collection:/);
+  });
+
+  test('successful getCollection for pools', async () => {
```

Comment:
Similar to my comments in `packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts`, we can unify tests that only differ based on whether they use groups or pools and possibly parameterize the tests based on `CollectionType` values.